dist/rise-cache.js

```diff
@@ -49,53 +49,47 @@ RiseVision.Common.RiseCache = (function () {
     }
 
     function fileRequest(isCacheRunning) {
-      var xhr = new XMLHttpRequest(),
-        url, str, separator, request;
+      var url, str, separator;
 
       if (isCacheRunning) {
         // configure url with cachebuster or not
         url = (nocachebuster) ? BASE_CACHE_URL + "?url=" + encodeURIComponent(fileUrl) :
           BASE_CACHE_URL + "cb=" + new Date().getTime() + "?url=" + encodeURIComponent(fileUrl);
 
-        // custom request object to provide in response
-        request = {
-          xhr: xhr,
-          url: url
-        };
-
-        xhr.open("GET", url, true);
-
-        xhr.addEventListener("loadend", function () {
-          var status = xhr.status || 0;
-
-          if (status >= 200 && status < 300) {
-            callback(request);
-          } else {
-            callback(request, new Error("The request failed with status code: " + status));
-          }
-        });
-
-        xhr.send();
-
+        makeRequest("GET", url);
```

Comment:
@donnapep I was think, should this also try a HEAD request as out rise cache accepts it? cheers